errolyn/a10.py

```diff
@@ -0,0 +1,35 @@
+
+bit_string = input("Please enter a 32-bit integer: ") # example: 10000000111111110000001100001111
+# there is no input sanitization or error handling since the assignment did not ask for it,
+# I would turn this into it's own function if that was a requirement.
+bit_array = [] # holds the raw bit sets in an array
+bit_parce =[] # holds parced values of bit_array
+
+def split_string(string):
+    while string: # this terminates when there is nothing left in "string".
+        bit_array.append(string[:8]) #takes the first 8 charaters and appends to array
+        string = string[8:] # removes first 8 charaters
+
+def parce_8_bit_string():
```

Comment:
`parse` is the correct spelling for this word.

Reply:
Also, from the prompt, I suspect the instructor wants a function with the signature along the lines of:

```python
def bitstring_to_int(bitstring):
    return bitstring_as_an_int_value
```